lib/ruby_lsp/check_docs.rb

```diff
@@ -94,13 +94,21 @@ def run_task
             # end
             # ```
           DOCS
-        elsif !/\[.* demo\]\(.*\.gif\)/.match?(documentation) || !gif_exists?(file_path)
+        elsif !/\[.* demo\]\(.*\.gif\)/.match?(documentation)
           T.must(missing_docs[class_name]) << <<~DOCS
             Missing demonstration GIF. Each request and extension must be documented with a GIF that shows the feature
             working. For example:
 
             # [Inlay hint demo](../../inlay_hint.gif)
           DOCS
+        elsif !gif_exists?(file_path)
+          T.must(missing_docs[class_name]) << <<~DOCS
+            The GIF for the request documentation does not exist. Make sure to add it in the `misc/` folder,
```

Comment:
Can we use the full path here instead of the `misc` folder? We're exporting this rake task to allow Ruby LSP extensions to use it and we want to allow them to be flexible about where they store their GIFs.

Reply:
Hello, I'm not sure to understand what you mean by `full path`? As we provide flexibility on where extension store their GIFs, maybe the best way is to not specify path at all? I did a try here: https://github.com/Shopify/ruby-lsp/pull/749/commits/31694e549fafa73dbd4bd9dbbd486aae2560d166

Tell me if I'm wrong, I can fix it. 👍